### DISCUSSION TITLE
[React] Using `autoUpdate` with `useClientPoint`

### DISCUSSION THREAD

**Discussion by Doesntmeananything** [floating-ui/floating-ui#2246] [Q&A] [answered]
Hi! Thank you for such an awesome library.

I have what might be a weird use case: 
1. I would like to position a floating element relative to the mouse cursor coordinates when clicking the reference element
2. After which I would like `autoUpdate` to handle the position of the floating element when scrolling the ancestor container

I've found a clean implementation of behavior 1 via `useClientPoint`, where I just disable it when the floating element is rendered. However, `useClientPoint` seems to conflict with `autoUpdate` even when disabled. Only removing `useClientPoint` from interactions entirely restores the correct `autoUpdate` behavior.

Is there any other way to have behavior 1 and 2 working together?

Here's a codesandbox based on the reusable Popover example with this scenario:
https://codesandbox.io/s/relaxed-brown-9t6gy5?file=/src/Popover.tsx

**Comment by atomiks** [comment, marked as answer]
This was asked as a question, but it does look like an issue to me. It works for touch devices but didn't account for this use case for non-touch devices. Thanks for the report.

**Reply by Doesntmeananything** [reply]
For posterity: this is fixed in https://github.com/floating-ui/floating-ui/pull/2249. Thanks very much for such a quick fix!
